packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -84,6 +84,12 @@ export class VaTextInput {
     | 'text'
     | 'url';
 
+  /**
+   * The step attribute is a number, or the string 'any', that specifies the granularity of the value. For example: `<va-text-input type="number" step=".1"/>` enables float/decimal values to be valid and increment by one-tenth. <br/>
```

Comment:
This is really great info for the prop. Will Stencil let us link to that [MDN article](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/step) you have in the PR description by chance? 🤔

Reply:
it will, I added the link